advanced/Scripts/webpage.sh

```diff
@@ -110,7 +110,7 @@ SetWebPassword() {
         # Prevents a bug if the user presses Ctrl+C and it continues to hide the text typed.
         # So we reset the terminal via stty if the user does press Ctrl+C
         trap '{ echo -e "\nNo password will be set" ; stty sane ; exit 1; }' INT
-        read -s -p "Enter New Password (Blank for no password): " PASSWORD
+        read -s -p -r "Enter New Password (Blank for no password): " PASSWORD
```

Comment:
This needs to be `read -r -s -p "Enter New Password (Blank for no password): " PASSWORD`

`-p` takes a string as prompt argument.